homework.md

```diff
@@ -4,110 +4,120 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2*3*6*1*10\34?
 
-1322
+10.588
 
-## Question 4 
+## Question 4
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hypertext Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Making a website beautiful
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
 ```css
-#h1 {
-
+h1 {
 }
 ```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language used in web development
```

Comment:
Can you explain more for me, please?

Reply:
JavaScript is usually a scripting language for interacting with web pages.

Reply:
JavaScript is a programming language that can provide dynamic interfacing to websites.

Reply:
good?